pubspec.yaml

```diff
@@ -1,34 +1,8 @@
-name: maplibre_gl
-description: A Flutter plugin for integrating MapLibre Maps inside a Flutter application on Android, iOS and web platforms.
-version: 0.19.0+2
-repository: https://github.com/maplibre/flutter-maplibre-gl
-issue_tracker: https://github.com/maplibre/flutter-maplibre-gl/issues
+name: flutter_maplibre_gl
```

Comment:
```suggestion
name: flutter_maplibre_gl
repository: https://github.com/maplibre/flutter-maplibre-gl
issue_tracker: https://github.com/maplibre/flutter-maplibre-gl/issues
publish_to: 'none'
```

Reply:
I would change the name here to `maplibre_gl_workspace`

Reply:
Updated the suggestion.